### PR TITLE
Use fin-hypegrid-event-logger in demo

### PR DESCRIPTION
**Pull request by joneit**
[`fin-hypegrid-event-logger`](https://github.com/fin-hypergrid/event-logger) is a new plugin that uses my new [`starlog`](https://www.npmjs.com/package/starlog) event logger, something I was playing with over break.

The logging output (when `demo.vent = true`) is basically the same as it was.

This PR:
* Loads the plugin and connects it to `demo.vent`.
* Moves `if (vent) { ... }` custom logging code to the plug-in ([here](https://github.com/fin-hypergrid/event-logger/blob/master/custom-listeners.js)).
* Changes selection array event properties (`.rows`, `.columns`, and `.selections`) to getters so they are not executed unless actually referenced.

### CODE REVIEW

**Comment by joneit** [comment]
@nmichaud Good point. I rolled it back here and will submit a new PR for lazy event props.

**Comment by joneit** [comment]
Reviewers @nmichaud @dcchuck @Dwaynekj do feel free to comment on /event-logger repo (via new issue](https://github.com/fin-hypergrid/event-logger/issues)).

File: src/Hypergrid.js [hunk 1]
```diff
@@ -89,7 +91,7 @@ var Hypergrid = Base.extend('Hypergrid', {
         );
 
         // Install shared plug-ins (those with a `preinstall` method)
-        Object.getPrototypeOf(this).installPlugins(options.plugins);
+        Hypergrid.prototype.installPlugins(options.plugins);
```
Comment (nmichaud) [inline src/Hypergrid.js]: This change seems unrelated to the PR?

Comment (joneit) [reply src/Hypergrid.js]: that is true.... it is unimportant & trivial & need to put it somewhere

Comment (joneit) [reply src/Hypergrid.js]: @nmichaud aside from this are you ready to approve?

**Comment by joneit** [comment]
btw, lazy event props were moved to new PR #671

**Comment by nmichaud** [comment]
Yeah looks good to me
